src/components/button_group.scss

```diff
@@ -2,23 +2,29 @@
   display: inline-flex;
   align-items: center;
 
-  .btn, .btn-primary, .btn-secondary, .btn-delete {
-    &:first-child:not(:last-child) {
-      border-top-right-radius: 0;
-      border-bottom-right-radius: 0;
+  .btn,
+  .btn-primary,
+  .btn-secondary,
+  .btn-delete {
+    + .btn,
+    + .btn-primary,
+    + .btn-secondary,
+    + .btn-delete {
+      margin-left: 1px;
```

Comment:
@Jeremy-Walton Thinking we should add a token for 1px. For example: `--rm-space-4x-small: calc(var(--rm-space-scale-unit) * 0.1); // 1px`

Though I'm also curious why we need a 1px margin here.

Reply:
This PR is purely making what we already have compliant with formatting standards. If you want to rewrite this, it should be done as a separate task than the linting

The 1px is to adjust for overlapping the borders (or box shadows. I forget exactly).